lib/index.js

```diff
@@ -104,6 +59,25 @@ const listBackups = async (TableName) => {
   return data.BackupSummaries || [];
 };
 
+/**
+ *
+ * @param tables
+ * @returns {Promise<void>}
+ */
+const removeStaleBackup = async (tables) => {
+  tables.map(async (tableName) => {
+    const backupSummaries = await listBackups(tableName);
+
+    backupSummaries.map(async (table) => {
+      const params = {
+        BackupArn: table.BackupArn,
+      };
+      await Dynamodb.deleteBackup(params).promise();
+      console.log(`Deleted Backups done for ${tableName}`);
```

Comment:
All `console.log` should use a prefix like `@unly/serverless-plugin-db-backups - `